app/js/api/index.js

```diff
@@ -15,6 +15,7 @@ class APIRoot {
     this.Buy = new API("stock_transactions/buy");
     this.Sell = new API("stock_transactions/sell");
     this.Stock = new API("stocks");
+    this.Issues = new API("api/github_issues");
```

Comment:
network tab showed client hitting incorrect route than what worked directly via browser hitting the api.